pulp_ansible/app/galaxy/views.py

```diff
@@ -214,7 +216,7 @@ def get_queryset(self):
         return versions
 
 
-class GalaxyCollectionVersionDetail(DistributionMixin, views.APIView):
+class GalaxyCollectionVersionDetail(DistributionMixin, generics.GenericAPIView):
```

Comment:
Can you explain which facilities of GenericApiView we are using?

Reply:
I cannot really answer this question, other than the fact that with `views.APIView` I would get the error `AttributeError: 'super' object has no attribute 'get_serializer_context'`.  After reading the drf docs around `get_serializer_context`, I was led to understand that it was a function of generic views. As such, I swapped out the mixin, and got the expected outcome.

Reply:
Thank you!